src/pages/bestStories/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import axios from "axios";
+import CONSTANTS from "../../constants/appConstant";
+import Story from "./../../components/Story/index";
+
+const { BASE_API_URL } = CONSTANTS;
+export default class TopStories extends Component {
+  state = {
+    topStories: [],
+    isLoading: true,
+  };
+
+  componentDidMount() {
+    this.fetchTopStories();
+  }
+
+  fetchTopStories = async () => {
+    try {
+      const { data: topStoriesIds } = await axios.get(
```

Comment:
Hi sarita,
you are doing good. let's follow best practice for writing axios call from now on okay ?

Here's my guide :

for using axios, 

create a http.js file inside utils folder.

![image](https://user-images.githubusercontent.com/22271897/121465570-c24d6080-c9d5-11eb-9a7b-4c2cffd8dfa0.png)

keep this in http.js file,
```suggestion
import axios from "axios";

import configs from "../config";

const http = axios.create({
  headers: {
    "Content-Type": "application/json"
  },
  baseURL: configs.baseURI
});

export default http;```

Reply:
Now perform axios call inside service folder

```suggestion
    import config from "../config";
import http from "../utils/http";

const { api } = config;

/**
 * Function to create a comment
 * 
 * @param data any
 * @param id string
 */
export async function createNewComment(data: any, id: string) {
  const url = `${api.endpoints.posts + "/" + id + api.endpoints.comments}`;
  const response = await http.post(url, data);
 
  return response;
}
```

![image](https://user-images.githubusercontent.com/22271897/121465746-e8730080-c9d5-11eb-95f1-c95de27e60ed.png)

Reply:
and use it like this:
![image](https://user-images.githubusercontent.com/22271897/121466060-7a7b0900-c9d6-11eb-8988-3e7171b7f332.png)

Reply:
config file
![image](https://user-images.githubusercontent.com/22271897/121467115-43a5f280-c9d8-11eb-92c4-aafc0fe98fbb.png)